zenoh-protocol/tests/channel.rs

```diff
@@ -305,6 +305,27 @@ fn channel_udp() {
     });
 }
 
+#[cfg(all(feature = "unix", target_family = "unix"))]
+#[test]
+fn channel_unix() {
+    let _ = std::fs::remove_file("/tmp/socket.sock");
+    // Define the locator
+    let locators: Vec<Locator> = vec!["unix//tmp/socket.sock".parse().unwrap()];
```

Comment:
Although /tmp is available in the totality of the system, I believe that using the current directory for the unix socket path is a better approach:
1. It is guaranteed that the cargo test will have write permissions while executing the test
2. System files are not created outside the test directory

I also suggest to use a more explanatory name for the socket, e.g., zenoh-test-unix-socket.sock